### DISCUSSION TITLE
TLS certificate validation not implemented

### DISCUSSION THREAD

**Discussion by MugiGumi** [DesktopECHO/Pi-hole-for-Android#75] [Q&A] [answered]
Anyway to fix this error? https://ibb.co/YXcGFb2

**Comment by DesktopECHO** [comment, marked as answer]
If you can, try a different WiFi network.  

You can also download the file using the browser installed on the device.  When the image is downloaded, open Pi Deploy and tap Properties.  Change the _Source path_ from:

`https://github.com/DesktopECHO/Pi-hole-for-Android/releases/latest/download/raspbian.tgz`

..to the location of the downloaded file (exact path may vary by device):

`/sdcard/Download/raspbian.tgz `

**Reply by MugiGumi** [reply]
I did managed to get it working by just downloading the 64bit image directly. Thanks for the quick reply! 
![Screenshot_20231124-023126](https://github.com/DesktopECHO/Pi-hole-for-Android/assets/67396833/33a7734c-a15f-4e39-a4f9-56f461a9e95e)

**Comment by aluoliv** [comment]
I had the same error, I changed  the path, but now I have a new error:

![Imagem do WhatsApp de 2023-12-25 à(s) 23 29 06_ce7b06c6](https://github.com/DesktopECHO/Pi-hole-for-Android/assets/154852084/752d519d-609f-4e0e-910f-1b83d8ae56d1)

**Reply by DesktopECHO** [reply]
What is the make and model of the device?

**Reply by aluoliv** [reply]
Its a TV BOX with processor amlogic S905X3 and Android 9  (it is rooted).

Model: T95 mas Plux  (same this [link](https://www.turewell.com/products/android-9-tv-box-4gb-ram-32gb-rom-t95-max-tv-box-amlogic-s905x3-quad-core-cortex-a55-media-player-with-8k-bt4-0-2-4g-5-0ghz-dual-band-wifi))

**Reply by DesktopECHO** [reply]
I believe those Android boxes ship with a 32-bit kernel. 

Go to _**Container Properties**_ -> _**Source Path**_ 

Set the image to download the 32-bit image instead:

`https://github.com/DesktopECHO/Pi-hole-for-Android/releases/latest/download/raspbian32.tgz` (Instead of _raspbian.tgz_)

Then try to re-deploy the the image.
